static/src/javascripts/projects/common/modules/commercial/dfp/private/render-advert.js

```diff
@@ -31,12 +31,28 @@ define([
      *
      */
 
+    function addClassIfHasClass(newClassNames) {
```

Comment:
would it be worth naming each of these functions to make it clearer what each is doing?